10-Types-of-contributions.md

```diff
@@ -0,0 +1,228 @@
+# Types of contributions
+In this chapter, we will discuss the types of contributions that open-source projects involve.
+
+Mostly, there are two types of contributions. Which are;
+
+- Non-coding type of contribution
+
+- Coding type of contribution
+
+   ## A. Non-Coding Type Of Contributions
+  
+  Non-coding contributions do not involve writing or modifying code. Non-coding contributions are essential for the sustainability and growth of open-source projects. They allow individuals with a variety of skills and backgrounds to participate and make meaningful contributions to the software and the community.
+
+ ### Common types of non-coding contributions
+ 
+  - Documentation
+  - Graphic design
+  - Blog writing
+
+
+### Documentation
+
+Are you a lover of words? Do grammar errors irritate you, like seeing pineapple on pizza? Then, documentation is for you! 
```

Comment:
This is great. Let's also add the following after the pineapple line:

Do you like breaking down complex problems or helping people learn?